### DISCUSSION TITLE
How to display nz-table no result template outside of the table?

### DISCUSSION THREAD

**Discussion by newshahn** [NG-ZORRO/ng-zorro-antd#7963] [General]
So when you use `nzNoResult` property like this:

```
<nz-table
  nzShowSizeChanger
  [nzData]="listOfData"
  [nzFrontPagination]="false"
  [nzLoading]="loading"
  [nzLoadingIndicator]="loaderTemplate"
  [nzNoResult]="noResultsTemplate"
>
```

The noResultTemplate will be shown inside the table. I want it to show **instead** of the table and in the place of the table. Is that possible?

**Comment by PishoLu** [comment]
you mean <nz-table *ngIf="listOfData.length;else noResultsTemplate">?

**Comment by HyperLife1119** [comment]
Do you mean you want to use the empty component inside the table component? It has a different style than the regular empty component.

```html
<nz-embed-empty nzComponentName="table"></nz-embed-empty>
```
